bindings/python/iota_sdk/types/address.py

```diff
@@ -59,6 +63,40 @@ class NFTAddress:
     type: int = field(default_factory=lambda: int(AddressType.NFT), init=False)
 
 
+@json
+@dataclass
+class ImplicitAccountCreationAddress:
+    """Represents an implicit account creation address that can be used to transition an account.
+    Attributes:
+        address: The hex encoded Ed25519 Address.
+    """
+    address: Ed25519Address
```

Comment:
Kind of the same question as JS, what will happen to the JSON serialisation?

Reply:
no idea here, also interested 👀 
probably fine though, as python doesnt have a concept of types itll just get forced in there

Reply:
![mhm](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fi.pinimg.com%2Foriginals%2F6d%2F8d%2F32%2F6d8d32fb6583dbe375c37e71d84e6c0b.gif&f=1&nofb=1&ipt=33c16f288a6949c101343c476d76f53ef56064488ab3ce669207dd5c3c43ac89&ipo=images)

Reply:
Tbh I don't know why it would be fine, just seems to me that's it's going to scope 🤷🏻